testing/functional/tests/cypress.json

```diff
@@ -16,5 +16,6 @@
   },
   "video": false,
   "chromeWebSecurity": false,
-  "defaultCommandTimeout": 10000
+  "defaultCommandTimeout": 10000,
+  "numTestsKeptInMemory": 0
```

Comment:
Was it changed due to high memory consumption?

Reply:
https://github.com/bcgov/SIMS/pull/1151#discussion_r848475821